src/Symfony/Component/Routing/Router.php

```diff
@@ -411,4 +413,15 @@ private function getConfigCacheFactory()
 
         return $this->configCacheFactory;
     }
+
+    private function checkDeprecatedOption($key)
```

Comment:
```suggestion
    private function checkDeprecatedOption(string $key): void
```

Reply:
that would break the current code for non-string options

Reply:
Oh, okay, didn't know they exist. Let's add only `: void` then?

Reply:
to me, `void` is useless, especially on a private method...